Shared/Extensions/URLExtensions.swift

```diff
@@ -360,10 +360,11 @@ extension URL {
 
 extension URL {
     public var isErrorPageURL: Bool {
-        if let host = self.host {
-            return self.scheme == "http" && host == "localhost" && path == "/errors/error.html"
-        }
-        return false
+        return scheme == "http" && host == "localhost" && path.contains("/errors/")
+    }
+    
+    public var safeBrowsingErrorURL: Bool {
```

Comment:
Isn't this exact same as `isErrorPageURL`? If so, maybe remove, or else maybe just return `isErrorPageURL`?

Reply:
No there is `errors/error.html` for network and certificate errors and `errors/safeBrowsingError.html` for phishing errors only.

We need to check for `safeBrowsingError` separately because we want to enable Brave shields for this page and we don't want to do that for regular error pages.

I could do some refactor or add comments to make it more clear

And `isErrorPageURL` helper applies to both `error` and `safeBrowsingError`

Reply:
But the logic is literally identical:
`return scheme == "http" && host == "localhost" && path.contains("/errors/")`
Is something missing in the `path`? Maybe a copy/paste mistake?

Reply:
Yes I copied it and forgot to change, sorry, good catch